package.json

```diff
@@ -7,15 +7,17 @@
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
     "start": "npm run dev",
-    "lint": "eslint --ext .js,.vue src",
+    "lint": "eslint --fix --ext .js,.vue src",
```

Comment:
Don't change the script. If you plan to use --fix, do it locally on your machine, because this test runs on Travis CI and if there are linting issues which are present but fixed due to --fix, the tests should fail but they won't. But the actual code won't change. So those linting issues will remain in the code.